lib/public/components/common/popover/PopoverEngine.js

```diff
@@ -151,7 +154,7 @@ class PopoverEngine {
         displayZoneMargins,
         position,
         projectors,
-        setChildrenSize,
+        configuration,
     ) {
```

Comment:
If `configuration` is optional, it should be handled if undefined/null is provided

Reply:
Here it's just transfered and the constructor accept undefined, so it's fine here

Reply:
constructor doesn't accept undefined because it will throw `Uncaught TypeError: Cannot read properties of undefined (reading 'resetSize')`

Reply:
Indeed, nice catch!